dist/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+<script defer src="index.bundle.js"></script><script defer src="print.bundle.js"></script></head>
+
+<body>
+    <header>
+        <h1>Leaderboard </h1>
+    </header> 
+
+    <main>
+        <div id="recent-scores">
+            <div class="top">
+                <h2>Recent scores</h2>
+                <button type="button" id="refresh-button">Refresh</button>
+            </div>
+            <ul id="scores-list"></ul>
+        </div>
+
+        <div id="form-container">
+            <h2>Add your score</h2>
+            <form action="submit" id="scores-form">
+                <input type="text" id="score-name-input" placeholder="Your name" required>
+                <input type="text" id="score-number-input" placeholder="Your score" required>
+                <input type="button" id="add-score" value="Submit">
+            </form>
```

Comment:
[OPTIONAL] For some reason, the validations are not working as expected. I believe you might have more events that are required. Since it's a form, with "submit" event should be enough to add the scores to the list. If you try the "click" or "on key press" events, you might need custom validations done through JavaScript.

You can see some invalid scores submitted in this screenshot:
![image](https://user-images.githubusercontent.com/79114233/151267741-e23c277d-4f76-4564-835d-549530869830.png)